blog/release-v1.13/index.mdx

```diff
@@ -0,0 +1,104 @@
+---
+authors:
+- KubeEdge SIG Release
+categories:
+- General
+- Announcements  
+date: 2023-01-23
+draft: false
+lastmod: 2023-01-23
+summary: KubeEdge v1.13 is live! 
+tags:
+- KubeEdge
+- kubeedge
+- edge computing
+- kubernetes edge computing
+- K8s edge orchestration
+- edge computing platform
+- cloud native
+- iot
+- iiot
+- release v1.13
+- v1.13
+title: KubeEdge v1.13 is live!
+---
+
+On January 23, 2023, KubeEdge released v1.13. The new version introduces several enhanced features, significantly improving performance, security, and edge device management.
```

Comment:
```suggestion
On Jan 18, 2023, KubeEdge released v1.13. The new version introduces several enhanced features, significantly improving performance, security, and edge device management.
```
It seems that this date is the official release time of release 1.13  https://github.com/kubeedge/kubeedge/commit/65522a5c0f5d8b4b7e9d083b7afa72924090d3a0

Reply:
made the change, sorry for missing it.